### ISSUE TITLE
Incorrect layout in this .docx file

### ISSUE THREAD

**Issue by parrenin** [ONLYOFFICE/DocumentServer#1568]
### This issue is unique.

- [X] I have used the [search tool](https://github.com/ONLYOFFICE/DesktopEditors/issues?q=) and did not find an issue describing my bug.

### Operating System

Linux (DEB package)

### Version information

6.4.2.6

### Expected Behavior

The attached .docx file should display the same way in OO and Word 2019.

[bug-OO-margin.docx](https://github.com/ONLYOFFICE/DesktopEditors/files/7792734/bug-OO-margin.docx)

### Actual Behavior

The file has 3 pages in OO, and only 2 in Word.
This seems to be related to the top margin of page 2.

### Reproduction Steps

1. Open the .docx file in OO => the document has 3 pages
2. Open the same file in Word 2019 => the document has only 2 pages

**Comment by ShockwaveNN** [comment]
Moving this issue to DocumentServer repo, since it's our base product and problem actual there too

**Comment by parrenin** [comment]
Still present in OO 7.0.0

**Comment by ShockwaveNN** [comment]
Yes, we will close issue as soon as our dev team is fixed it, but it's not fixed

**Comment by parrenin** [comment]
Still repro in 7.0.2.
